data/modules/SearchRescue/SearchRescue.lua

```diff
@@ -1813,6 +1813,10 @@ local onEnterSystem = function (player)
 	 mission.target = createTargetShip(mission)
       end
    end
+	  
+      for _,discarded_ship in pairs(discarded_ships) do
```

Comment:
This is just a [belt & braces](http://www.phrases.org.uk/meanings/61250.html) approach, did it just in case.